### ISSUE TITLE
TestRunner doesn't know that ays crashed

### ISSUE THREAD

**Issue by ashraffouda** [Jumpscale/ays9#360]
#### Installation information

- jumpscale version: 9.2.0
```
from ays_testrunner.testrunner import AYSTestRunnerFactory 
runner = AYSTestRunnerFactory.get(name='custom', config={'bp_paths': ["/opt/code/github/jumpscale/ays9/tests/bp_test_templates/core/test_longjobs.yaml"]})
 runner.run()
```
```
[Mon30 15:12] - testrunner.py       :195 :j.aystestrunner.custom         - INFO     - Collecting tests from paths ['/opt/code/github/jumpscale/ays9/tests/bp_test_templates/core/test_longjobs.yaml']
[Mon30 15:12] - testrunner.py       :545 :j.aystestrunner.custom         - INFO     - Running test test_longjobs.yaml
AYS testrunner results
---------------------------

Total number of tests: 1
Number of passed tests: 1
Number of failed/error tests: 0
```
it shows that all tests passed but, I checked ays and found it crashed because of a syntax error

**Comment by ahussein** [comment]
do you mean that the server was down but still the runner said that the test passed ?
the runner assumes that the ays server is running, it will not start a new server. This means that if the AYS server is already running the server will not start new instance. If you changed something in the ays code itself you need to restart AYS yourself.

**Comment by ashraffouda** [comment]
what happened is that AYS server crashed, and the test didn't know about it, 
I think some thing like this the execution of the test should raise an exception that ays server is not running or something like htis

**Comment by ahussein** [comment]
fixed by #365
